src/core/Akka.Tests/Pattern/BackoffSupervisorSpec.cs

```diff
@@ -90,15 +91,30 @@ public static Props Props(IActorRef probe)
         private IActorRef Create(BackoffOptions options) => Sys.ActorOf(BackoffSupervisor.Props(options));
         #endregion
 
-        [Fact(Skip = "Racy on Azure DevOps")]
+        [Fact]
         public void BackoffSupervisor_must_start_child_again_when_it_stops_when_using_Backoff_OnStop()
         {
-            var supervisor = Create(OnStopOptions());
-            supervisor.Tell(BackoffSupervisor.GetCurrentChild.Instance);
-            var c1 = ExpectMsg<BackoffSupervisor.CurrentChild>().Ref;
+            var supervisor = Create(OnStopOptions().WithManualReset());
```

Comment:
Supervisor props need to be forced to use manual reset to make sure that it doesn't auto reset the internal restart count mid-spec and fails the spec.